bos_mint/config-defaults.yaml

```diff
@@ -3,15 +3,16 @@ debug: False
 project_name: MINT
 project_sub_name: The BOS Manual Intervention Module
 secret_key: # enter any random string 
+advanced_features: True
```

Comment:
this should not appear in default config

Reply:
fixed